src/cms/components/parks/page-header.json

```diff
@@ -0,0 +1,26 @@
+{
+  "collectionName": "components_parks_page_headers",
+  "info": {
+    "name": "PageHeader",
+    "icon": "archway",
+    "description": ""
+  },
+  "options": {},
+  "attributes": {
+    "PageTitle": {
```

Comment:
Will update. Note that after component fields are defined, they can be "configured" to add labels and descriptions that are separate from the field names. Adding ticket so that in test & prod, those are configured to be helpful to editors.